ruby_connectors/spec/spec_helper.rb

```
@@ -7,6 +7,20 @@
 # Load environment variables from .env file
 Dotenv.load
 
+VCR.configure do |config|
+  config.cassette_library_dir = "spec/cassettes"
+  config.hook_into :webmock
+  config.configure_rspec_metadata!
+  config.filter_sensitive_data("<GITHUB_ACCESS_TOKEN>") { ENV.fetch("GITHUB_ACCESS_TOKEN", nil) }
+  config.filter_sensitive_data("<MOTHERDUCK_TOKEN>") { ENV.fetch("MOTHERDUCK_TOKEN", nil) }
+  
+  vcr_mode = ENV['VCR_MODE'] =~ /rec/i ? :all : :once
```

rubocop

[convention] Use `match?` instead of `=~` when `MatchData` is not used.

(Performance/RegexpMatch)